packages/console/src/pages/ApiResourceDetails/components/DeleteForm/index.tsx

```diff
@@ -0,0 +1,84 @@
+import React, { useState } from 'react';
+import { toast } from 'react-hot-toast';
+import { useTranslation } from 'react-i18next';
+import { useNavigate } from 'react-router-dom';
+
+import Button from '@/components/Button';
+import Card from '@/components/Card';
+import CardTitle from '@/components/CardTitle';
+import TextInput from '@/components/TextInput';
+import Close from '@/icons/Close';
+import api from '@/utilities/api';
+
+import * as styles from './index.module.scss';
+
+type Props = {
+  id: string;
+  name: string;
+  onClose: () => void;
+};
+
+const DeleteForm = ({ id, name, onClose }: Props) => {
+  const { t } = useTranslation(undefined, { keyPrefix: 'admin_console' });
+
+  const navigate = useNavigate();
+
+  const [inputName, setInputName] = useState<string>('');
+  const [isDeleting, setIsDeleting] = useState(false);
+
+  const inputMismatched = inputName !== name;
+
+  const handleDelete = async () => {
+    try {
+      setIsDeleting(true);
+      await api.delete(`/api/resources/${id}`);
+      onClose();
+      navigate(`/api-resources`);
+      toast.success(t('api_resource_details.api_resource_deleted', { name }));
+    } catch (error: unknown) {
+      console.error(error);
+    } finally {
+      setIsDeleting(false);
+    }
+  };
+
+  return (
+    <Card className={styles.card}>
+      <div className={styles.headline}>
+        <CardTitle title="api_resource_details.reminder" />
+        <Close className={styles.close} onClick={onClose} />
```

Comment:
better make it a IconButton?

Reply:
<img width="797" alt="image" src="https://user-images.githubusercontent.com/10806653/158515738-9cb6b2ed-ee33-455b-bac5-e1a7723a1405.png">

Reply:
Per slack discussion, could leave a todo here just make it easy to track and update

Reply:
LOG-1890